{{cookiecutter.project_slug}}/clients/vue3/src/views/Login.vue

```diff
@@ -1,27 +1,34 @@
 <template>
-  <div class="login">
-    <form @submit.prevent="attemptLogin()">
+  <div
+    class="login container w-full sm:max-w-prose content-center mx-auto py-5 bg-gray-100 rounded-lg"
```

Comment:
**Info:** Currently using the same `container` styling + formatting for both the `Login` view and the `Signup` view, so this list of TailwindCSS class names could be pulled out into a reusable global class for containers to be used app-wide.

Reply:
Yep let's pull it out and do that
Let's also remove any unused classes